src/frontend/apps/desk/src/features/teams/components/Member/MemberGrid.tsx

```diff
@@ -78,8 +84,11 @@ export const MemberGrid = ({ teamId, currentRole }: MemberGridProps) => {
             headerName: t('Emails'),
           },
           {
-            field: 'role',
+            id: 'role',
```

Comment:
Reading this update, I wonder if ID on custom cell should be set to a unique value per row. Same question applies for the avatar column. A unique ID is a general good practice to avoid unnecessary re-render, wdyt? I read Cunningham documentation, and found no more info about it.
Let's keep it as it is. Nice one !!

Reply:
The `id` seems to be only for the column, so it should be ok:
https://github.com/openfun/cunningham/blob/30ec253ab0b7f655f90152f3f0797a8493f0ee2b/packages/react/src/components/DataGrid/utils.tsx#L27